gui/window_character_finder.gui

```diff
@@ -72,6 +72,28 @@ window = {
 								datamodel_wrap = 7
 								maxverticalslots = 2
 							}
+
+							blockoverride "character_age" {
+								#Warcraft
+								text_single	 = {
+									visible = "[GreaterThan_int32(Character.GetBirthDate.GetYear, '(int32)20')]"
+									name = "character_age"
+									text = " [Character.GetAge]"
+									default_format = "#low"
+									fontsize = 15
+									align = nobaseline
+								}
+
+								#Warcraft
+								text_single	 = {
+									visible = "[LessThanOrEqualTo_int32(Character.GetBirthDate.GetYear, '(int32)20')]"
+									name = "character_age_2"
+									text = " ???"
+									default_format = "#low"
+									fontsize = 15
+									align = nobaseline
+								}
+							}
```

Comment:
I don't think you need this part if you changed `lists.gui`.

Reply:
> 
> 
> I don't think you need this part if you changed `lists.gui`.

That's probably true. Changed that part before I changed lists.gui